openslides/users/config_variables.py

```diff
@@ -26,6 +26,7 @@ def get_config_variables():
     yield ConfigVariable(
         name='users_pdf_welcometitle',
         default_value='Welcome to OpenSlides',
+        input_type='markupText',
```

Comment:
Please remove this line. We have to add html parsing support first for access data pdf. And markup text it's not really required for this pdf.